styles/pages/TorchboxAcademy.module.scss

```diff
@@ -0,0 +1,49 @@
+.container {
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+.textContainer {
+    margin: 0 auto $grid * 3 auto;
+    max-width: 755px;
+    padding: 0 $grid;
+
+    @include media-query(medium) {
+        margin: 0 auto $grid * 4 auto;
+    }
+
+    @include media-query(large) {
+        padding: 0;
+    }
+}
+
+.mainContentContainer {
+    margin: 0 auto $grid * 3 auto;
+    max-width: 784px;
```

Comment:
**Suggestion:** not really a suggestion but something to keep in mind.

You could add a local SCSS variable to this file i.e. $extra-text-padding;
`max-width: $content--narrow + $extra-text-padding;`
This one might not be ideal here but can be useful if you have attributes that depends on the same size of another locally, such as when we indent content to the same size as the padding.

Round the padding down and use grid (might not fit the designs but could be better for consistency)
`max-width: $content--narrow + $grid;`

Or
`max-width: $content--narrow + 29px;`

Reply:
Thank you, that's good to know! I will try this out here 👍